### PR TITLE
Restyle inquiries & requests

### PR DESCRIPTION
**Pull request by emilyzhong**
## Restyle inquiries & requests
https://app.asana.com/0/860297512787618/1113269590853597/f
Restyled inquiries and requests based on figma wireframes

### Related PRs
None

### Migrations
None

### Tests Performed, Edge Cases
None

### Screenshots
image
image

CC: @by-co

### CODE REVIEW

File: app/javascript/components/commissions/Commissions.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  render() {
+    if (!this.state.componentDidMount) {
+      return <LoadingOverlay fullPage={true} />;
+    }
+    return (
+      <div className="mw7 center">
+        <h1>Inquiries</h1>
+        {
+          this.state.commissions.map((commission) => {
+            return <Inquiry commission={commission} />
```
Comment (by-co) [inline app/javascript/components/commissions/Commissions.jsx]: Add `key={ }` prop to remove console warnings c:

File: app/javascript/components/requests/Receipt.jsx [hunk 2]
```diff
@@ -66,47 +39,89 @@ class Receipt extends React.Component {
...
+            <h5>{isRental ? "Rental Start" : ""}</h5>
+            <p>{startDate}</p>
+          </div>
+          <div className="pa3 flex-grow-1">
+            <h5>{isRental ? "Rental End" : ""}</h5>
+            <p>{endDate}</p>
+          </div>
+          <div className="pa3">
+            <h5>Price</h5>
+            <p>{"$" + convertToCurrency(this.state.request.receipt.price)}</p>
```
Comment (by-co) [inline app/javascript/components/requests/Receipt.jsx]: For some reason it's only showing 1 digit after the decimal, e.g. `$100.0` 🤔

File: app/javascript/components/requests/Receipt.jsx [hunk 3]
```diff
@@ -66,47 +39,89 @@ class Receipt extends React.Component {
...
+          </div>
+          <div className="pa3">
+            <h5>Date Placed</h5>
+            <p>{new Date(this.state.request.created_at).toLocaleDateString()}</p>
+          </div>
+          <div className={classNames("relative", "mh3", {"requests-dropdown-selected" : this.state.dropDownVisible})}>
+            <button 
+              onClick={() => this.setState({ dropDownVisible: !this.state.dropDownVisible })}
+              className="request-ellipsis ml3 self-start br-100 pa0 pointer bn outline-0">
```
Comment (by-co) [inline app/javascript/components/requests/Receipt.jsx]: tachyons 😍

File: app/javascript/components/requests/Receipt.jsx [hunk 4]
```diff
@@ -16,42 +17,14 @@ class Receipt extends React.Component {
     };
   }
 
-  getAttr = (request) => {
-    let attr = {
-      "Price": "$" + convertToCurrency(request.receipt.price),
-      "Purchase Date": new Date(request.receipt.purchase_date).toLocaleDateString(),
-      "Transaction Type": request.receipt.transaction_type,
-      "Description": request.receipt.comment
```
Comment (by-co) [inline app/javascript/components/requests/Receipt.jsx]: Can we bring back `request.receipt.comment` somewhere? Maybe underneath the patron's message?

File: app/javascript/components/requests/Request.jsx [hunk 5]
```diff
@@ -31,105 +33,99 @@ class Request extends React.Component {
...
+        <div className="ttu b mt3 ml3 f6">
+          {this.renderStatus()}
+        </div>
+        <div className="flex justify-between items-start pr5 pa3">
+          <div className="flex">
+            <Touchable onPress={() => this.navigateToWork(this.state.request.work.id)}>
+              <div className="w4 pb6 relative mr3">
+                <img className="work-image fit-cover w-100 h-100 pointer absolute" src={this.state.request.work.featured_image.url} />
+              </div>
+            </Touchable>
```
Comment (by-co) [inline app/javascript/components/requests/Request.jsx]: Maybe wrap the touchable around title + media also?

File: app/javascript/components/requests/Receipt.jsx [hunk 6]
```diff
@@ -66,47 +39,89 @@ class Receipt extends React.Component {
...
+        <div className="ttu b mt3 ml3 f6">
+          { this.renderStatus() }
+        </div>
+        <div className="flex justify-between items-start pr5 pa3">
+          <div className="flex">
+            <Touchable onPress={() => this.navigateToWork(this.state.request.work.id)}>
+              <div className="w4 pb6 relative mr3">
+                <img className="work-image fit-cover w-100 h-100 pointer absolute" src={this.state.request.work.featured_image.url} />
+              </div>
+            </Touchable>
```
Comment (by-co) [inline app/javascript/components/requests/Receipt.jsx]: Same here, wrap around title+media so the whole section is clickable?

**Comment by by-co** [comment]
Oops I meant to mark as "requested changes" but whatever

File: app/javascript/components/requests/Receipt.jsx (hunk 4 above)
Comment (emilyzhong) [reply app/javascript/components/requests/Receipt.jsx]: image
 something like this?

File: app/javascript/components/requests/Receipt.jsx (hunk 2 above)
Comment (emilyzhong) [reply app/javascript/components/requests/Receipt.jsx]: Fixed in `currency.js`!

File: app/javascript/components/requests/Receipt.jsx (hunk 4 above)
Comment (by-co) [reply app/javascript/components/requests/Receipt.jsx]: yes plese
